support-frontend/assets/pages/digital-subscription-checkout/components/digitalPaymentTerms.jsx

```diff
@@ -4,19 +4,18 @@ import { FormSection } from 'components/checkoutForm/checkoutForm';
 import { type Option } from 'helpers/types/option';
 import type { PaymentMethod } from 'helpers/paymentMethods';
 import { DirectDebit } from 'helpers/paymentMethods';
-import styles from './summary.module.scss';
-import EndSummary from 'pages/digital-subscription-checkout/components/endSummary/endSummary';
 import DirectDebitTerms from 'components/subscriptionCheckouts/directDebitTerms';
 
 export default function DigitalPaymentTerms(props: {
   paymentMethod: Option<PaymentMethod>,
 }) {
   return (
-    <FormSection>
-      <div className={styles.endSummay}>
-        <EndSummary />
-      </div>
-      {(props.paymentMethod === DirectDebit) && <DirectDebitTerms />}
-    </FormSection>
+    <span>
```

Comment:
Do we want to return an empty span here for non-DD payment methods?

Reply:
You do have to return jsx or null. I guess I could do a ternary and return null otherwise?

Reply:
Would be cleaner I think

Reply:
Could you move the conditional out of this component into the parent:

```jsx
{props.paymentMethod === DirectDebit && <DigitalPaymentTerms />}
```

You could then simplify this `DigitalPaymentTerms` component by removing the `paymentMethod` prop, and you could also remove the wrapping `<span>` ?

Reply:
That would be nicer all other things being equal, but the parent is currently more of a composer of he pieces rather than a manager of presentation logic, so I'd slightly prefer to leave it in the sub-component.